rust/ballista/rust/Cargo.toml

```diff
@@ -25,6 +25,6 @@ members = [
     "scheduler",
 ]
 
-[profile.release]
-lto = true
-codegen-units = 1
+#[profile.release]
```

Comment:
@Dandandan This was an accidental commit. I had to comment this out so that I could build and test without really slow build times. Is there a better way for me to work around this?

Reply:
It would really be convenient for this to have this feature https://github.com/rust-lang/cargo/issues/6988 as I agree if you just want to run it with "reasonable" performance it shouldn't take ages to compile Ballista. In my experience it roughly 2x as long for project to build with lto (maybe worse when you compare it with incremental builds).

It is possible to do via flags as well, but earlier this didn't work because of the structure of the Ballista projects (multiple binaries per crate as far as I remember), maybe we can just temporary remove this (and be a bit slower) and see if we can enable it in a different way.
Another route I saw is just appending `lto = true` in a build script when creating binaries.